src/test/java/test/LicenseTest.java

```diff
@@ -45,4 +45,17 @@ void bitbucket_license_secret_name(Product product) throws Exception {
                 .getEnv()
                 .assertHasSecretRef("SETUP_LICENSE", "license_secret", "mykey");
     }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
```

Comment:
We didn't have Jira license unit tests so I didn't need to remove anything but we didn't also have Bamboo test, which we support. I have added that one.